e2e_samples/fabric_dataops_sample/infra/setup-infra.sh

```diff
@@ -1,257 +1,65 @@
 #!/bin/bash
-source .env
+
+#######################################################
+# Deploys all necessary azure and Fabric resources
+#
+# Prerequisites:
+# - User is logged in to the azure cli
+# - Correct Azure subscription is selected
+#######################################################
+
+source ./.env
 
 ## Environment variables
-base_name="$BASE_NAME"
-location="$LOCATION"
-adls_gen2_connection_id="$ALDS_GEN2_CONNECTION_ID"
+base_name="$TF_VAR_base_name"
+location="$TF_VAR_location"
+adls_gen2_connection_id="$ADLS_GEN2_CONNECTION_ID"
+tenant_id="$FABRIC_TENANT_ID"
 
 # Variable set based on Terraform output
 tf_storage_account_name=""
 tf_storage_container_name=""
 tf_storage_account_url=""
-tf_fabric_capacity_name=""
 
 # Fabric bearer token variables, set globally
 fabric_bearer_token=""
 fabric_api_endpoint="https://api.fabric.microsoft.com/v1"
 
 # Fabric related variables
-fabric_workspace_name="ws-$base_name"
-onelake_name="onelake"
-lakehouse_name="lh_main"
 alds_gen2_shortcut_name="sc_adls_gen2_main"
 alds_gen2_shortcut_path="Files"
-environment_name="env_$base_name"
-environment_desc="Default Fabric environment for '$base_name' project"
-lakehouse_desc="Lakehouse for '$base_name' project"
-notebook_names=("nb-city-safety" "nb-covid-data")
-pipeline_names=("pl-covid-data")
 
 deploy_terraform_resources() {
     cd "$1" || exit
 
-    user_principal_name=$(az ad signed-in-user show --query userPrincipalName -o tsv)
+    #user_principal_name=$(az account show --query user.name -o tsv)
+    user_principal_type=$(az account show --query user.type -o tsv)
+    if [[ "$user_principal_type" == "user" ]]; then use_cli="true"; else use_cli="false"; fi
+    echo "[DEBUG] use_cli is ${use_cli}"
+
 
     terraform init
     terraform apply \
         -auto-approve \
-        -var "base_name=$base_name" \
-        -var "location=$location" \
-        -var "fabric_capacity_admin=$user_principal_name"
+        -var "use_cli=$use_cli" #\
```

Comment:
Please review the parameters passed to terraform. Do we want to pass the "capacity admin" as well?

Reply:
same as above, please review the newly committed `.envtemplate` file and let me know

Reply:
Fixed